app/mailers/patient_mailer.rb

```diff
@@ -186,11 +187,15 @@ def assessment_email(patient)
       format.html { render layout: 'main_mailer' }
     end
     patient.active_dependents_and_self.each { |pat| add_success_history(pat) }
+  # This method is called in in the main loop of the send_assessments_job
+  # It is important to capture and log all errors and let the loop continue to send assessments
   rescue StandardError => e
```

Comment:
It sounds like we could just change the language to unexpected error instead of network error and get the same end-user result (an error occurred and there's nothing you can do about it) We are logging the full error to Sentry so there is no need from our side to have the history message be more specific.

Reply:
Oh that's fine with me. I wasn't sure if that language specifically was important to the PO team.